src/Job/Message.php

```diff
@@ -119,11 +119,18 @@ public function getCallable()
      */
     public function getArgument($key = null, $default = null)
     {
+        // support old jobs that still have args
+        if (array_key_exists('args', $this->parsedBody)) {
+            $data = $this->parsedBody['args'][0];
+        } else {
+            $data = $this->parsedBody['data'];
+        }
```

Comment:
What about new messages that are consumed by old workers? Wouldn't they still be missing their arguments?

Reply:
That is true. I guess for a short time we will need to double up the args and data in the payload until we remove args. I pushed an update.